doc/user/quick-sql-grammar.md

```diff
@@ -271,7 +271,7 @@ setting to override this default.
 **Possible Values**: `11g`, `12c`, `19c`, `21c`, `23c`
 **Default Value**: `19c`
 
-Specifies the database version the syntax should be compatible with.
+Specifies the database version the syntax should be compatible with. Only version number is significant (and not the postfix). Therefore, 21 and 23.1.1 are also legitimate values.
```

Comment:
The regular expression only checks for 2 digit groups version version therefore 23.1.1 will only take 23.1

Reply:
Simplified the major version parsing.